src/measure_theory/prod.lean

```diff
@@ -0,0 +1,689 @@
+/-
+Copyright (c) 2020 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import measure_theory.giry_monad
+import measure_theory.set_integral
+
+/-!
+# The product measure
+
+In this file we define and prove properties about the binary product measure. If `α` and `β` have
+σ-finite measures `μ` resp. `ν` then `α × β` can be equipped with a σ-finite measure `μ.prod ν` that
+satisfies `(μ.prod ν) s = ∫⁻ x, ν (prod.mk x ⁻¹' s) ∂μ`.
```

Comment:
I think that this form is more readable for a mathematician who doesn't remember what is `prod.mk` in Lean.
```suggestion
satisfies `(μ.prod ν) s = ∫⁻ x, ν {y | (x, y) ∈ s} ∂μ`.
```